scss/_nav.scss

```diff
@@ -188,7 +188,28 @@
   > .tab-pane {
     display: none;
   }
+
+  > .tab-pane-with-nested-tab {
+    display: none;
+    margin: subtract(-$spacer, -$nav-tabs-border-width);
+  }
+
   > .active {
     display: block;
   }
 }
+
+.nested-tab-content {
```

Comment:
Can't we remove this rule and use in the HTML `tab-content border-0`? The rules seem to be the same.

Reply:
Yes we can, but, then there will be a slight difference of vertical alignment. With `tab-content border-0` instead of `nested-tab-content`, the content text (i.e. "Content of Link A") is no more left aligned with the nested tab title "Link A" and the tab title "Tab 1" which are both aligned.
We have no illustration of that content to know if it should be aligned of not, so I've asked the design.

Reply:
Ah yes my bad, I haven't seen the minus in the `substract` at first. So maybe just add this rule in `.nested-tab-content` and use both classes in the HTML to avoid redifining `.tab-pane` and `.active`.

Reply:
At last, I found out that I could remove totally this class.